autobotAI_integrations/integrations/nessus/__init__.py

```
@@ -0,0 +1,175 @@
+
+        client = NessusClient(
+            url=connection_details.get("url"),
+            access_key=connection_details.get("access_key"),
+            secret_key=connection_details.get("secret_key"),
+            verify_ssl=connection_details.get("verify_ssl", False),
+        )
+
+        return SDKClient(client=client)
+
+    @classmethod
+    def test_connection(cls, connection_details: Union[RestAPICreds, SDKCreds]) -> bool:
```

Ruff (0.15.18)

[warning] Use `X | Y` for type annotations

Convert to `X | Y`

(UP007)